packages/wethegit-components/src/components/accordion/component/AccordionItem.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useRef } from "react"
+
+import { Icon, IconDefs, IconSymbol, Tag, Text } from "@local/components"
+
+import styles from "../accordion.module.scss"
+
+const DEFAULT_TAG = "h3" as const
+
+export function GenericSection({ ...props }) {
+  const { as = DEFAULT_TAG, ...rest } = props
+
+  return <Tag as={as} {...rest} />
+}
+
+export interface CardsData {
+  title: string
+  text: string | object
+}
+
+interface Props {
+  handleToggle: (index: number) => void
+  active: number | null
+  card: CardsData
+  icon: string
+  index: number
+}
+
+const AccordionItem: React.FC<Props> = ({ index, active, handleToggle, card, icon }) => {
+  const contentEl = useRef<HTMLDivElement>(null)
+  const { title, text } = card
+
+  if (!card) return null
+
+  return (
+    <div
+      className={styles.accordionPanel}
+      aria-expanded={active === index ? "true" : undefined}
+    >
+      <Tag as={DEFAULT_TAG} id={`accordionHeading${index}`}>
+        <button
+          className={`${active === index ? styles.active : ""}`}
+          onClick={() => handleToggle(index)}
+          tabIndex={0}
+          aria-controls={`accordionBody${index}`}
+        >
+          <span className={styles.accordionTitle}>{title}</span>
+
+          <IconDefs>
+            <IconSymbol id={icon} size={27}>
+              <path
+                d="M6 9L12 15L18 9"
+                stroke="#000000"
+                strokeWidth="1.2"
+                strokeLinecap="round"
+                strokeLinejoin="round"
+              ></path>
+            </IconSymbol>
+          </IconDefs>
+
+          <Icon id={icon} className={styles.accordionIcon} />
+        </button>
+      </Tag>
+
+      <Tag
```

Comment:
This can be just a `div` no need to use `Tag` as it won't be replaced by a dynamic value.